code/game/machinery/requests_console.dm

```diff
@@ -249,6 +249,7 @@ var/list/obj/machinery/requests_console/allConsoles = list()
 		for(var/mob/M in player_list)
 			if(!istype(M,/mob/new_player) && M.client)
 				M << "<b><font size = 3><font color = red>[department] announcement:</font color> [message]</font size></b>"
+				world << sound('sound/misc/notice.ogg')

```

Comment:
Not to mention that this would play the sound globally for EVERY MOB IN THE PLAYERLIST.

Deafening cacophony of music.